src/nft/neonft.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
please add a "shell-ext" folder under src and put the NFT extension project in that folder